edit/linter-report.js

```diff
@@ -0,0 +1,165 @@
+/* global linter editors clipString createLinterHelpDialog makeSectionVisible */
+'use strict';
+
+// eslint-disable-next-line no-var
+var linterReport = (() => {
+  const cms = new Map();
```

Comment:
So it contains tables? in that case it should be named accordingly.
Maybe uhm... `tables`? :-)